src/profile/v100/app_preferences/gateways/gateway.rs

```diff
@@ -137,8 +137,8 @@ impl Gateway {
 }
 
 impl Gateway {
-    pub fn wellknown() -> Vec<Self> {
-        vec![Self::mainnet(), Self::stokenet()]
+    pub fn wellknown() -> Gateways {
+        Gateways::from_iter(vec![Self::mainnet(), Self::stokenet()])
```

Comment:
this can be written just ` Gateways::from_iter([Self::mainnet(), Self::stokenet()])` (no need for `vec!`)